vpc-bastion.tf

```diff
@@ -17,6 +17,7 @@ resource "google_compute_subnetwork" "vpc_subnetwork_bastion" {
 
 # TODO Peerings and SSH
 resource "google_compute_network_peering" "peering_osd_to_bastion" {
+  count                               = var.osd_gcp_private ? 1 : 0
```

Comment:
also the vpcs from the bastion and the bastion itself needs to have this conditional right? afaik are only needed when the osd_gcp_private is enabled, in public deployments is not needed.